src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
For the case of a synthetic event created via `jQuery.Event`, there won't be an `originalEvent` property. For a native event created that way, [one will be there](https://github.com/gabrielschulhof/jquery/blob/2826dfc65b6c9b9968be3297e6ae01cfced521c8/src/event.js#L643) and it might get used when it shouldn't.

That said, I don't see a fail in the compat branch for this so I'm going to try and create one.